engine/src/main/resources/org/hibernate/validator/ValidationMessages_fr.properties

```diff
@@ -1,53 +1,53 @@
-javax.validation.constraints.Min.message             = doit \u00EAtre au minimum \u00E9gal \u00E0 {value}
-javax.validation.constraints.Negative.message        = doit \u00EAtre strictement n\u00E9gatif
-javax.validation.constraints.NegativeOrZero.message  = doit \u00EAtre n\u00E9gatif ou \u00E9gal \u00E0 0
-javax.validation.constraints.NotBlank.message        = ne peut pas \u00EAtre vide
-javax.validation.constraints.NotEmpty.message        = ne peut pas \u00EAtre vide
-javax.validation.constraints.NotNull.message         = ne peut pas \u00EAtre nul
-javax.validation.constraints.Null.message            = doit \u00EAtre nul
-javax.validation.constraints.Past.message            = doit \u00EAtre dans le pass\u00E9
-javax.validation.constraints.PastOrPresent.message   = doit \u00EAtre dans le pass\u00E9 ou dans le pr\u00E9sent
-javax.validation.constraints.Pattern.message         = doit respecter "{regexp}"
-javax.validation.constraints.Positive.message        = doit \u00EAtre strictement positif
-javax.validation.constraints.PositiveOrZero.message  = doit \u00EAtre positif ou \u00E9gal \u00E0 0
-javax.validation.constraints.Size.message            = la taille doit \u00EAtre comprise entre {min} et {max}
+javax.validation.constraints.AssertFalse.message     = doit avoir la valeur false
+javax.validation.constraints.AssertTrue.message      = doit avoir la valeur true
+javax.validation.constraints.DecimalMax.message      = doit \u00eatre inf\u00e9rieur \u00e0 ${inclusive == true ? ' ou \u00e9gal \u00e0 ' : ''}{value}
+javax.validation.constraints.DecimalMin.message      = doit \u00eatre sup\u00e9rieur \u00e0 ${inclusive == true ? ' ou \u00e9gal \u00e0 ' : ''}{value}
+javax.validation.constraints.Digits.message          = valeur num\u00e9rique hors limites (<{integer} chiffres>.<{fraction} chiffres> attendu)
+javax.validation.constraints.Email.message           = doit \u00eatre une adresse \u00e9lectronique syntaxiquement correcte
+javax.validation.constraints.Future.message          = doit \u00eatre une date dans le futur
+javax.validation.constraints.FutureOrPresent.message = Doit \u00eatre une date dans le pr\u00e9sent ou le futur
+javax.validation.constraints.Max.message             = doit \u00eatre inf\u00e9rieur ou \u00e9gal \u00e0 {value}
+javax.validation.constraints.Min.message             = doit \u00eatre sup\u00e9rieur ou \u00e9gal \u00e0 {value}
+javax.validation.constraints.Negative.message        = doit \u00eatre inf\u00e9rieur \u00e0 0
+javax.validation.constraints.NegativeOrZero.message  = doit \u00eatre inf\u00e9rieur ou \u00e9gal \u00e0 0
+javax.validation.constraints.NotBlank.message        = ne doit pas \u00eatre blanc
+javax.validation.constraints.NotEmpty.message        = ne doit pas \u00eatre vide
+javax.validation.constraints.NotNull.message         = ne doit pas avoir la valeur NULL
+javax.validation.constraints.Null.message            = doit avoir la valeur NULL
```

Comment:
I don't think `la valeur NULL` is an improvement. I can fix it myself as it's the French translation.